src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -26,48 +26,17 @@ export const evidenceTypesDescription = ({ formData }) => {
 };
 
 
-// Shows or collapses the "Which should I choose?" link at the bottom of the evidence types page
-// TODO: Investigate whether this should just use `expandUnder`
-export class EvidenceTypeHelp extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      expanded: false
-    };
-  }
-
-  toggleOpen = (e) => {
-    e.preventDefault();
-    this.setState({ open: !this.state.open });
-  }
-
-  expandedContent = (
-    <div>
-      <h3>Types of evidence</h3>
-      <h4>VA medical records</h4>
-      <p>If you were treated at a VA medical center or clinic, you have VA medical records. This includes Tri-Care.</p>
-      <h4>Private medical records</h4>
-      <p>If you were treated by a private doctor, including Veteran’s Choice, we will need to see those records to proceed with your claim. Like DBQs.</p>
-      <h4>Lay statements or other evidence</h4>
-      <p>Also known as "buddy statements," written accounts from family or other people who know you can help support a claim. In most cases your medical records are enough, but claims involving Post Traumatic Stress Disorder or Military Sexual Trauma sometimes benefit from Lay Statements.</p>
-      <button className="va-button-link" onClick={this.toggleOpen}>Close</button>
-    </div>
-  )
-
-  render() {
-    const className = classNames(
-      'form-expanding-group',
-      { 'form-expanding-group-open': this.state.open }
-    );
-
-    return (
-      <div className={className}>
-        <button className="va-button-link dashed-underline" onClick={this.toggleOpen}>Which should I choose?</button>
-        {this.state.open && this.expandedContent}
-      </div>
-    );
-  }
-}
+export const evidenceTypeHelp = (
+  <InformationLink>
+    <h3>Types of evidence</h3>
+    <h4>VA medical records</h4>
+    <p>If you were treated at a VA medical center or clinic, you have VA medical records. This includes Tri-Care.</p>
+    <h4>Private medical records</h4>
+    <p>If you were treated by a private doctor, including Veteran’s Choice, we will need to see those records to proceed with your claim. Like DBQs.</p>
```

Comment:
@bethpotts @peggygannon @gnakm should we spell out acronyms or is this instance (DBQ) self-evident?

Reply:
@cvalarida @cehsu this is not the most updated content. I had made edits to the prototype of all of these screens you reference in the PR.